qa-portal-angular/projects/qa-cv/src/app/view-cv/services/view-cv.service.ts

```diff
@@ -11,47 +11,47 @@ import { QaHttpService } from 'projects/portal-core/src/app/_common/services/qa-
 export class ViewCvService {
 
 
-  constructor(private qaHttp: QaHttpService) {
-  }
-
-  /** GET cv by id. Will 404 if id not found */
-  getCvForId(id: string): Observable<ICvModel> {
-    return this.qaHttp.get<ICvModel>({ ref: 'GET_CV_FOR_ID', params: { id: id.toString() }}).pipe(
-      take(1)
-    );
-  }
-
-  getSkillsForTrainee(): Observable<any> {
-    return this.qaHttp.get<any>({ ref: 'GET_SKILLS_FOR_TRAINEE'}).pipe(
-      take(1)
-    );
-  }
-
-  getCurrentCvForTrainee(): Observable<ICvModel> {
-    return this.qaHttp.get<ICvModel>({ ref: 'GET_CURRENT_CV'}).pipe(
-      take(1)
-    );
-  }
-
-  displayPdf(cv: ICvModel) {
-    return this.qaHttp.post<any>({ ref: 'GENERATE_CV'}, cv).subscribe((response) => {
-      const file = new Blob([response], {type: APPLICATION_PDF_CONTENT_TYPE});
-      const fileURL = URL.createObjectURL(file);
-      window.open(fileURL, '_blank');
-    });
-  }
-
-  // /** POST: add a new cv to the server */
-  createCv(cv: ICvModel): Observable<ICvModel> {
-    return this.qaHttp.post<ICvModel>({ ref: 'SAVE_CV_DATA'}, cv).pipe(
-      take(1)
-    );
-  }
-
-  /** PUT: update the cv on the server */
-  updateCv(cv: ICvModel): Observable<ICvModel> {
-    return this.qaHttp.put<ICvModel>({ ref: 'SAVE_CV_DATA'}, cv).pipe(
-      take(1)
-    );
-  }
+//   constructor(private qaHttp: QaHttpService) {
```

Comment:
Cleanup this file, there's a lot of commented out code.

Reply:
qa-portal-angular/projects/qa-cv/src/app/view-cv/services/view-cv.service.ts
is deleted on this branch as it is not needed. 
I don't understand why this has appeared in the merge.